src/inlined.h

```diff
@@ -44,6 +51,9 @@ static inline uint32_t simple_uhash(const char *name) {
     return hval;
 }
 
+/**
+ * ktsaou: Your help needed
```

Comment:
calculate a simple hash, but at the same time compare the string to be hashed with another.

So, normally one would do:

```c
hash = simple_hash(string);
if(hash == myhash && strcmp(string, "mystring") == 0) ...
```

The above requires 2 loops on `string`.

So, `if(simple_hash_strcmp(string, "mystring", &hash) == 0) ...` merges these 2 loops into one.

Currently it is only used for the `SET` line in pluginsd.c (the most common line send by external plugins to netdata).